Gemfile

```diff
@@ -13,8 +13,9 @@ gem 'coffee-rails', '~> 4.0.0'
 # See https://github.com/sstephenson/execjs#readme for more supported runtimes
 # gem 'therubyracer',  platforms: :ruby
 
-gem 'dotenv'
-gem 'letsfreckle-client', branch: :master, require: 'letsfreckle', github: 'ryanlecompte/letsfreckle-client'
+gem 'dotenv-rails'
+gem 'freckles', github: 'ombulabs/freckles',
+                branch: 'add-last-page-to-responses-with-rel-last-link'
```

Comment:
@mauro-oto Could we use this project now? https://github.com/timcraft/freckles

I think that your PR got implemented in `master`